cmd/openshift-install/main.go

```diff
@@ -33,6 +34,10 @@ func main() {
 			assetStock.MasterIgnition(),
 			assetStock.WorkerIgnition(),
 		}
+	case manifestsCommand.FullCommand():
+		targetAssets = []asset.Asset{
+			assetStock.Operators(),
```

Comment:
Manifests

Reply:
Just change the top level function call or the entire package naming?

Reply:
I'd prefer the entire package be renamed.

Reply:
@crawford Done. If this all looks good to you, I can squash the commits once more for the merge.